### ISSUE TITLE
Feature Request: Scheduled Tasks - Task run at client specific time

### ISSUE THREAD

**Issue by DrunkMunki** [prom3theu5/iNGEN-Ark-RCON-Desktop#8]
Hello,
i was wondering if you could put in a feature so i can send a broadcast message at a scheduled time on my PC?

the reason is i have a server reboot scheduled, some users forget about it and would like to send a broadcast at ... for example 4:55am (reboot is at 5am)

**Comment by prom3theu5** [comment]
Yeah sure - I could add the option into the scheduler thats there to launch a task at a certain time - no problem :)

**Comment by DrunkMunki** [comment]
any chance on an update on this feature?
